client/dashboard/sites/deployments/dataviews/fields.tsx

```diff
@@ -138,9 +142,22 @@ export function useDeploymentFields( {
 				getValue: ( { item } ) => {
 					return item.created_on;
 				},
-				render: ( { item } ) => {
-					return <TimeSince timestamp={ item.created_on } />;
-				},
+				render: ( { item } ) => (
+					<div>
+						<Tooltip
+							text={ getDeploymentTimeTooltipText(
+								locale,
+								item.completed_on,
+								item.created_on,
+								item.status
+							) }
+						>
+							<span>
```

Comment:
Question: Why do we need a `<span/>` here instead of directly using `<TimeSince/>`?